idea/src/org/jetbrains/kotlin/idea/inspections/KotlinRedundantOverrideInspection.kt

```diff
@@ -69,6 +68,15 @@ class KotlinRedundantOverrideInspection : AbstractKotlinInspection(), CleanupLoc
                 }
             }
 
+    private fun KtNamedFunction.containsOtherModifier(): Boolean {
+        MODIFIER_EXCLUDE_OVERRIDE.forEach {
```

Comment:
You can simplify this to `return MODIFIER_EXCLUDE_OVERRIDE.any { hasModifier(it) }`